synapse/module_api/callbacks/__init__.py

```diff
@@ -12,11 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from synapse.module_api.callbacks.account_validity_callbacks import (
-    AccountValidityModuleApiCallbacks,
-)
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from synapse.server import HomeServer
+
+from .account_validity_callbacks import AccountValidityModuleApiCallbacks
+from .spamchecker_callbacks import SpamCheckerModuleApiCallbacks
```

Comment:
I don't mind relative imports too much, but FWIW I don't think we use them. (Though if we care that much, we should enforce it; maybe https://github.com/PyCQA/isort/issues/1677 one day)

Reply:
Oops, that was [called out](https://github.com/matrix-org/synapse/pull/15237#discussion_r1134437204) in the previous PR and I forgot to update this commit in tandem. Thank you for catching!